runtime/gc_base/GCExtensions.cpp

```diff
@@ -235,7 +235,7 @@ MM_GCExtensions::updateIdentityHashDataForSaltIndex(UDATA index)
 }
 
 /*
- * computeDefaultMaxHeapForJava is for Java only, it will be called during gcParseCommandLineAndInitializeWithValues(),
+ * computeDefaultMaxHeapForJava is for Java8 only, it will be called during gcParseCommandLineAndInitializeWithValues(),
```

Comment:
- I don't understand reason for this change, it is not accurate to say this function set values for Java 8 only.
- As fas as reminding changes in this file are white spaces only this file should be excluded from this change set and not touched at all

Reply:
I have excluded the file from the commit.